Automattic/woocommerce-payments
# Making table rows "clickable", approach #2

Fixes #484
Alternative to https://github.com/Automattic/woocommerce-payments/pull/544 

#### Changes proposed in this Pull Request

- Create a new `ClickableCell` component to use for links within columns.
- Use the `ClickableCell` component within all cells, which are not linked yet.
- Adjust snapshots.

The `ClickableCell` component is a link with a particular class, which:

- Removes styles (color, text decoration, box shadow) in all states of the link.
- Uses -1 as `tabIndex` in order to be transparent to screen readers.
- Uses a negative margin to fill the expand the link within the whole cell, in combination with padding inside of the link to compensate.

#### Testing instructions

* Inspect each table (Transactions, Disputes, and Payouts) to make sure each column is clickable.
* Verify that ℹ️ links are still clickable.
